client/my-sites/upgrades/domain-management/dns/dns-record.jsx

```diff
@@ -1,7 +1,9 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
+import React from 'react';
+import classNames from 'classnames';
+import endsWith from 'lodash/string/endsWith';
```

Comment:
No need for lodash here: Use https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith